docs_src/microservices/device/V3Migration.md

```diff
@@ -271,15 +271,35 @@ TBD
 
 This section is specific to changes made only to **Device USB Camera**
 
-See [Top Level V3 Migration Guide](../../../V3TopLevelMigration) for details applicable to all EdgeX services and [All Device Services](#all-device-services) section above for details applicable to all EdgeX device services. . 
-
-TBD
+See [Top Level V3 Migration Guide](../../../V3TopLevelMigration) for details applicable to all EdgeX services and [All Device Services](#all-device-services) section above for details applicable to all EdgeX device services.
 
 #### Configuration
 
-TBD
+All USB camera rtsp streams need authentication by default. To properly configure credentials for the stream, follow the below instructions.
```

Comment:
Would it make more sense to only have this link to the official documentation to avoid having to maintain 2 copies of it?

ie. this migration guide will merely tell/warn the users about the change, and then link to the documentation regarding it.